backend/ops_api/tests/ops/notification/test_notification.py

```diff
@@ -110,11 +110,11 @@ def test_notification_creation(loaded_db, notification):
 
 @pytest.mark.usefixtures("app_ctx")
 def test_notifications_get_all(auth_client, loaded_db):
-    assert loaded_db.query(Notification).count() == 22
+    assert loaded_db.query(Notification).count() == 42
```

Comment:
test is flaky - would be better to get the count of the `Notification` in the DB and compare to the count returned in the response

Reply:
k, I'll make that change in the next round